.changeset/thin-pumpkins-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+fix: ensure `getInputFromAssetId` returns correct `InputMessage`
```

Comment:
I've been suggesting [shorter release notes](https://github.com/FuelLabs/fuels-ts/pull/2796#discussion_r1683652796) in some PRs.

I think you could almost swap the two, see what you think.

----

In this release, we:
 - Fixed the `getInputFromAssetId` method used in transaction summary operations

# Summary

Fixed an issue where `getInputFromAssetId` was returning incorrect `InputMessage`s based on base asset ids, resulting in empty operations being returned in the txn summaries.

Reply:
I've updated the description now based on [a9cb052](https://github.com/FuelLabs/fuels-ts/pull/2782/commits/a9cb0521cc414be8e7104ed5163fea48a3796e08) let me know your thoughts

Reply:
@maschad How about?

_made `getInputFromAssetId` return correct `baseAssetId` for a given `InputMessage`_

Reply:
Well @Torres-ssf it also applies to `InputCoins` as well, unfortunately the 72 character limitation isn't as expressive for cases like this 😢

Reply:
@maschad So `getInputFromAssetId` was returning the wrong `baseAssetId` not only for `InputMessage` but for `InputCoin` as well?

Reply:
Not exactly, but it was returning the `coinMessage` or `coinInput` as _undefined_ based on the incorrect `baseAssetId` being sent in some situations, I described in more detail [here](https://github.com/FuelLabs/fuels-ts/pull/2782#discussion_r1690388439)

Reply:
My main concern/suggestion was to make the `release notes` shorter:

Some that would fit like this:
![option-1](https://github.com/user-attachments/assets/4eafb8c8-ccd0-4021-9a8b-815e61f1b9a3)


Instead of this:
![option-2](https://github.com/user-attachments/assets/da0110cf-2035-4a0b-8599-d8eacd65a56b)

Reply:
7004319a6